### ISSUE TITLE
[Issue]: Cannot connect after upgrading to 10.9.0

### ISSUE THREAD

**Issue by Inovvia** [jellyfin/jellyfin#11534]
### Please describe your bug

I cannot access/connect. I even tried curling via localhost

### Reproduction Steps

1. I upgraded from 10.8.13
2. Upon starting 10.9.0 I get not response from the server

### Jellyfin Version

10.8.13

### if other:

10.9.0

### Environment

```markdown
- OS: Ubuntu 22.04.4 LTS
- Linux Kernel: 5.15.0-102-generic
- Virtualization: Docker
- FFmpeg Version: N/A
- Playback Method: N/A
- Hardware Acceleration: None
- GPU Model: None
- Plugins: AniList 7.0.0.0, AniSearch 4.0.0.0, Open Subtitles 19.0.0.0, TMDb 10.9.0.0, Studio Images 10.9.0.0, OMDb 10.9.0.0 MusicBrainz, 10.9.0.0 AudioDB, 10.9.0.0
- Reverse Proxy: nginx
- Base URL: /
- Networking: bridge
- Storage: cloud (local)
```

### Jellyfin logs

```shell
[20:00:25] [INF] [4] Main: Jellyfin version: 10.9.0
[20:00:25] [INF] [4] Main: Environment Variables: ["[JELLYFIN_WEB_DIR, /jellyfin/jellyfin-web]", "[JELLYFIN_CONFIG_DIR, /config/config]", "[JELLYFIN_CACHE_DIR, /cache]", "[JELLYFIN_FFMPEG, /usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_LOG_DIR, /config/log]", "[JELLYFIN_DATA_DIR, /config]"]
[20:00:25] [INF] [4] Main: Arguments: ["/jellyfin/jellyfin.dll", "--ffmpeg", "/usr/lib/jellyfin-ffmpeg/ffmpeg"]
[20:00:25] [INF] [4] Main: Operating system: Debian GNU/Linux 12 (bookworm)
[20:00:25] [INF] [4] Main: Architecture: X64
[20:00:25] [INF] [4] Main: 64-Bit Process: True
[20:00:25] [INF] [4] Main: User Interactive: True
[20:00:25] [INF] [4] Main: Processor count: 4
[20:00:25] [INF] [4] Main: Program data path: /config
[20:00:25] [INF] [4] Main: Log directory path: /config/log
[20:00:25] [INF] [4] Main: Config directory path: /config/config
[20:00:25] [INF] [4] Main: Cache path: /cache
[20:00:25] [INF] [4] Main: Web resources path: /jellyfin/jellyfin-web
[20:00:25] [INF] [4] Main: Application directory: /jellyfin/
[20:00:25] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Marking following migrations as applied because this is a fresh install: ["CreateNetworkConfiguration", "MigrateMusicBrainzTimeout", "MigrateNetworkConfiguration"]
[20:00:25] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /cache
[20:00:25] [INF] [4] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[20:00:25] [INF] [4] Jellyfin.Networking.Manager.NetworkManager: Defined LAN addresses: ["127.0.0.1/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
[20:00:25] [INF] [4] Jellyfin.Networking.Manager.NetworkManager: Defined LAN exclusions: []
[20:00:25] [INF] [4] Jellyfin.Networking.Manager.NetworkManager: Using LAN addresses: ["127.0.0.1/8", "10.0.0.0/8", "172.16.0.0/12", "192.168.0.0/16"]
[20:00:25] [INF] [4] Jellyfin.Networking.Manager.NetworkManager: Using bind addresses: ["127.0.0.1", "172.28.0.5"]
[20:00:25] [INF] [4] Jellyfin.Networking.Manager.NetworkManager: Remote IP filter is Allowlist
[20:00:25] [INF] [4] Jellyfin.Networking.Manager.NetworkManager: Filter list: []
[20:00:26] [INF] [4] Emby.Server.Implementations.ApplicationHost: There are pending EFCore migrations in the database. Applying... (This may take a while, do not stop Jellyfin)
[20:00:27] [WRN] [4] Microsoft.EntityFrameworkCore.Migrations: An operation of type 'SqlOperation' will be attempted while a rebuild of table 'ImageInfos' is pending. The database may not be in an expected state. Review the SQL generated by this migration to help diagnose any failures. Consider moving these operations to a subsequent migration.
[20:00:27] [WRN] [4] Microsoft.EntityFrameworkCore.Migrations: An operation of type 'SqlOperation' will be attempted while a rebuild of table 'ImageInfos' is pending. The database may not be in an expected state. Review the SQL generated by this migration to help diagnose any failures. Consider moving these operations to a subsequent migration.
[20:00:27] [WRN] [4] Microsoft.EntityFrameworkCore.Migrations: An operation of type 'SqlOperation' will be attempted while a rebuild of table 'Permissions' is pending. The database may not be in an expected state. Review the SQL generated by this migration to help diagnose any failures. Consider moving these operations to a subsequent migration.
[20:00:27] [WRN] [4] Microsoft.EntityFrameworkCore.Migrations: An operation of type 'SqlOperation' will be attempted while a rebuild of table 'Permissions' is pending. The database may not be in an expected state. Review the SQL generated by this migration to help diagnose any failures. Consider moving these operations to a subsequent migration.
[20:00:27] [WRN] [4] Microsoft.EntityFrameworkCore.Migrations: An operation of type 'SqlOperation' will be attempted while a rebuild of table 'Preferences' is pending. The database may not be in an expected state. Review the SQL generated by this migration to help diagnose any failures. Consider moving these operations to a subsequent migration.
[20:00:27] [WRN] [4] Microsoft.EntityFrameworkCore.Migrations: An operation of type 'SqlOperation' will be attempted while a rebuild of table 'Preferences' is pending. The database may not be in an expected state. Review the SQL generated by this migration to help diagnose any failures. Consider moving these operations to a subsequent migration.
[20:00:27] [WRN] [4] Microsoft.EntityFrameworkCore.Migrations: An operation of type 'SqlOperation' will be attempted while a rebuild of table 'Users' is pending. The database may not be in an expected state. Review the SQL generated by this migration to help diagnose any failures. Consider moving these operations to a subsequent migration.
[20:00:27] [WRN] [4] Microsoft.EntityFrameworkCore.Migrations: An operation of type 'SqlOperation' will be attempted while a rebuild of table 'Users' is pending. The database may not be in an expected state. Review the SQL generated by this migration to help diagnose any failures. Consider moving these operations to a subsequent migration.
[20:00:27] [INF] [4] Emby.Server.Implementations.ApplicationHost: EFCore migrations applied successfully
[20:00:28] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Saving system configuration
[20:00:28] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /cache
[20:00:28] [INF] [4] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: TMDb 10.9.0.0
[20:00:28] [INF] [4] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Studio Images 10.9.0.0
[20:00:28] [INF] [4] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: OMDb 10.9.0.0
[20:00:28] [INF] [4] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: MusicBrainz 10.9.0.0
[20:00:28] [INF] [4] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: AudioDB 10.9.0.0
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Marking following migrations as applied because this is a fresh install: ["DisableTranscodingThrottling", "CreateLoggingConfigHeirarchy", "MigrateActivityLogDatabase", "RemoveDuplicateExtras", "MigrateUserDatabase", "MigrateDisplayPreferencesDatabase", "RemoveDownloadImagesInAdvance", "MigrateAuthenticationDatabase", "FixPlaylistOwner", "MigrateRatingLevels"]
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Applying migration 'AddDefaultPluginRepository'
[20:00:28] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Saving system configuration
[20:00:28] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /cache
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Migration 'AddDefaultPluginRepository' applied successfully
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Applying migration 'ReaddDefaultPluginRepository'
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Migration 'ReaddDefaultPluginRepository' applied successfully
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Applying migration 'AddDefaultCastReceivers'
[20:00:28] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Saving system configuration
[20:00:28] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /cache
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Migration 'AddDefaultCastReceivers' applied successfully
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Applying migration 'UpdateDefaultPluginRepository10.9'
[20:00:28] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Saving system configuration
[20:00:28] [INF] [4] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /cache
[20:00:28] [INF] [4] Jellyfin.Server.Migrations.MigrationRunner: Migration 'UpdateDefaultPluginRepository10.9' applied successfully
[20:00:28] [INF] [4] Main: Kestrel is listening on 127.0.0.1
[20:00:28] [INF] [4] Main: Kestrel is listening on 172.28.0.5
[20:00:28] [WRN] [4] Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager: No XML encryptor configured. Key {461fb8d7-a2ac-475e-8a26-374129ca7af0} may be persisted to storage in unencrypted form.
[20:00:28] [WRN] [4] Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware: The WebRootPath was not found: /wwwroot. Static files may be unavailable.
[20:00:28] [INF] [4] Emby.Server.Implementations.ApplicationHost: Running startup tasks
[20:00:28] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: Daily trigger for Generate Trickplay Images set to fire at 2024-05-12 03:00:00.000 +00:00, which is 06:59:31.1048684 from now.
[20:00:28] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: Daily trigger for Extract Chapter Images set to fire at 2024-05-12 02:00:00.000 +00:00, which is 05:59:31.1006978 from now.
[20:00:29] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Found ffmpeg version 6.0.1
[20:00:29] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available decoders: ["libdav1d", "av1", "av1_cuvid", "av1_qsv", "h264", "h264_qsv", "h264_cuvid", "hevc", "hevc_qsv", "hevc_cuvid", "mpeg2video", "mpeg2_qsv", "mpeg2_cuvid", "mpeg4", "mpeg4_cuvid", "msmpeg4", "vc1_qsv", "vc1_cuvid", "vp8", "libvpx", "vp8_cuvid", "vp8_qsv", "vp9", "libvpx-vp9", "vp9_cuvid", "vp9_qsv", "aac", "ac3", "dca", "flac", "mp3", "truehd"]
[20:00:29] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available encoders: ["libsvtav1", "av1_nvenc", "av1_qsv", "av1_amf", "av1_vaapi", "libx264", "h264_amf", "h264_nvenc", "h264_qsv", "h264_v4l2m2m", "h264_vaapi", "libx265", "hevc_amf", "hevc_nvenc", "hevc_qsv", "hevc_vaapi", "mjpeg_qsv", "mjpeg_vaapi", "mpeg4", "msmpeg4", "libvpx", "libvpx-vp9", "aac", "libfdk_aac", "ac3", "alac", "dca", "flac", "libmp3lame", "libopus", "truehd", "libvorbis", "srt"]
[20:00:29] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available filters: ["deinterlace_qsv", "deinterlace_vaapi", "hwupload_cuda", "hwupload_vaapi", "libplacebo", "overlay_opencl", "overlay_qsv", "overlay_vaapi", "overlay_vulkan", "overlay_cuda", "procamp_vaapi", "scale_cuda", "scale_opencl", "scale_qsv", "scale_vaapi", "scale_vulkan", "tonemap_cuda", "tonemap_opencl", "tonemap_vaapi", "vpp_qsv", "yadif_cuda", "zscale", "alphasrc"]
[20:00:29] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available hwaccel types: ["cuda", "vaapi", "qsv", "drm", "opencl", "vulkan"]
[20:00:29] [INF] [4] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: /usr/lib/jellyfin-ffmpeg/ffmpeg
[20:00:29] [INF] [4] Emby.Server.Implementations.ApplicationHost: ServerId: f6e51edc4a9144e19856da5a22275510
[20:00:29] [INF] [4] Emby.Server.Implementations.ApplicationHost: Core startup complete
[20:00:29] [INF] [4] Main: Startup complete 0:00:04.6899722
[20:00:31] [INF] [6] Emby.Server.Implementations.ScheduledTasks.TaskManager: Clean up collections and playlists Completed after 0 minute(s) and 0 seconds
[20:00:33] [INF] [4] Emby.Server.Implementations.ScheduledTasks.TaskManager: Update Plugins Completed after 0 minute(s) and 1 seconds
[20:02:05] [INF] [25] Emby.Server.Implementations.Session.SessionManager: Sending shutdown notifications
[20:02:05] [INF] [28] Jellyfin.Networking.PortForwardingHost: Stopping NAT discovery
[20:02:05] [INF] [26] Main: Running query planner optimizations in the database... This might take a while
[20:02:05] [INF] [26] Emby.Server.Implementations.ApplicationHost: Disposing CoreAppHost
[20:02:05] [INF] [26] Emby.Server.Implementations.ApplicationHost: Disposing MusicBrainzArtistProvider
[20:02:05] [INF] [26] Emby.Server.Implementations.ApplicationHost: Disposing MusicBrainzAlbumProvider
[20:02:05] [INF] [26] Emby.Server.Implementations.ApplicationHost: Disposing PluginManager
[20:02:26] [INF] [1] Main: Jellyfin version: 10.9.0
[20:02:26] [INF] [1] Main: Environment Variables: ["[JELLYFIN_CACHE_DIR, /cache]", "[JELLYFIN_WEB_DIR, /jellyfin/jellyfin-web]", "[JELLYFIN_LOG_DIR, /config/log]", "[JELLYFIN_CONFIG_DIR, /config/config]", "[JELLYFIN_FFMPEG, /usr/lib/jellyfin-ffmpeg/ffmpeg]", "[JELLYFIN_DATA_DIR, /config]"]
[20:02:26] [INF] [1] Main: Arguments: ["/jellyfin/jellyfin.dll", "--ffmpeg", "/usr/lib/jellyfin-ffmpeg/ffmpeg"]
[20:02:26] [INF] [1] Main: Operating system: Debian GNU/Linux 12 (bookworm)
[20:02:26] [INF] [1] Main: Architecture: X64
[20:02:26] [INF] [1] Main: 64-Bit Process: True
[20:02:26] [INF] [1] Main: User Interactive: True
[20:02:26] [INF] [1] Main: Processor count: 4
[20:02:26] [INF] [1] Main: Program data path: /config
[20:02:26] [INF] [1] Main: Log directory path: /config/log
[20:02:26] [INF] [1] Main: Config directory path: /config/config
[20:02:26] [INF] [1] Main: Cache path: /cache
[20:02:26] [INF] [1] Main: Web resources path: /jellyfin/jellyfin-web
[20:02:26] [INF] [1] Main: Application directory: /jellyfin/
[20:02:27] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /cache
[20:02:27] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[20:02:27] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly Jellyfin.Plugin.AniDB, Version=7.0.0.0, Culture=neutral, PublicKeyToken=null from /config/plugins/AniDB_7.0.0.0/Jellyfin.Plugin.AniDB.dll
[20:02:27] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly Jellyfin.Plugin.AniList, Version=6.0.0.0, Culture=neutral, PublicKeyToken=null from /config/plugins/AniList_7.0.0.0/Jellyfin.Plugin.AniList.dll
[20:02:27] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly Jellyfin.Plugin.AniSearch, Version=4.0.0.0, Culture=neutral, PublicKeyToken=null from /config/plugins/AniSearch_4.0.0.0/Jellyfin.Plugin.AniSearch.dll
[20:02:27] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded assembly Jellyfin.Plugin.OpenSubtitles, Version=19.0.0.0, Culture=neutral, PublicKeyToken=null from /config/plugins/Open Subtitles_19.0.0.0/Jellyfin.Plugin.OpenSubtitles.dll
[20:02:27] [ERR] [1] Emby.Server.Implementations.Plugins.PluginManager: Failed to load assembly /config/plugins/TheTVDB_10.0.0.0/Jellyfin.Plugin.Tvdb.dll. This error occurs when a plugin references an incompatible version of one of the shared libraries. Disabling plugin
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Could not load type 'MediaBrowser.Common.Plugins.IPluginServiceRegistrator' from assembly 'MediaBrowser.Common, Version=10.9.0.0, Culture=neutral, PublicKeyToken=null'.
Could not load type 'MediaBrowser.Controller.Plugins.IServerEntryPoint' from assembly 'MediaBrowser.Controller, Version=10.9.0.0, Culture=neutral, PublicKeyToken=null'.
   at System.Reflection.RuntimeModule.GetTypes(RuntimeModule module)
   at System.Reflection.RuntimeModule.GetTypes()
   at Emby.Server.Implementations.Plugins.PluginManager.LoadAssemblies()+MoveNext()
System.TypeLoadException: Could not load type 'MediaBrowser.Common.Plugins.IPluginServiceRegistrator' from assembly 'MediaBrowser.Common, Version=10.9.0.0, Culture=neutral, PublicKeyToken=null'.
System.TypeLoadException: Could not load type 'MediaBrowser.Controller.Plugins.IServerEntryPoint' from assembly 'MediaBrowser.Controller, Version=10.9.0.0, Culture=neutral, PublicKeyToken=null'.
[20:02:27] [ERR] [1] Jellyfin.Networking.Manager.NetworkManager: Unable to parse bind override: https://jf.domain.tld/
[20:02:27] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN addresses: ["10.0.1.0/24"]
[20:02:27] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN exclusions: []
[20:02:27] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using LAN addresses: ["10.0.1.0/24"]
[20:02:27] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind addresses: []
[20:02:27] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Remote IP filter is Allowlist
[20:02:27] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Filter list: []
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: AniDB 7.0.0.0
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: AniList 7.0.0.0
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: AniSearch 4.0.0.0
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Open Subtitles 19.0.0.0
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: TMDb 10.9.0.0
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Studio Images 10.9.0.0
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: OMDb 10.9.0.0
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: MusicBrainz 10.9.0.0
[20:02:28] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: AudioDB 10.9.0.0
[20:02:29] [WRN] [1] Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware: The WebRootPath was not found: /wwwroot. Static files may be unavailable.
[20:02:29] [INF] [1] Emby.Server.Implementations.ApplicationHost: Running startup tasks
[20:02:29] [INF] [10] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /media/anime
[20:02:29] [INF] [11] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /media/movies
[20:02:29] [INF] [8] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /media/tv
[20:02:29] [INF] [1] Emby.Server.Implementations.ScheduledTasks.TaskManager: Daily trigger for Generate Trickplay Images set to fire at 2024-05-12 03:00:00.000 +00:00, which is 06:57:30.2458600 from now.
[20:02:29] [INF] [1] Emby.Server.Implementations.ScheduledTasks.TaskManager: Daily trigger for Extract Chapter Images set to fire at 2024-05-12 02:00:00.000 +00:00, which is 05:57:30.2344851 from now.
[20:02:29] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Found ffmpeg version 6.0.1
[20:02:29] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available decoders: ["libdav1d", "av1", "av1_cuvid", "av1_qsv", "h264", "h264_qsv", "h264_cuvid", "hevc", "hevc_qsv", "hevc_cuvid", "mpeg2video", "mpeg2_qsv", "mpeg2_cuvid", "mpeg4", "mpeg4_cuvid", "msmpeg4", "vc1_qsv", "vc1_cuvid", "vp8", "libvpx", "vp8_cuvid", "vp8_qsv", "vp9", "libvpx-vp9", "vp9_cuvid", "vp9_qsv", "aac", "ac3", "dca", "flac", "mp3", "truehd"]
[20:02:29] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available encoders: ["libsvtav1", "av1_nvenc", "av1_qsv", "av1_amf", "av1_vaapi", "libx264", "h264_amf", "h264_nvenc", "h264_qsv", "h264_v4l2m2m", "h264_vaapi", "libx265", "hevc_amf", "hevc_nvenc", "hevc_qsv", "hevc_vaapi", "mjpeg_qsv", "mjpeg_vaapi", "mpeg4", "msmpeg4", "libvpx", "libvpx-vp9", "aac", "libfdk_aac", "ac3", "alac", "dca", "flac", "libmp3lame", "libopus", "truehd", "libvorbis", "srt"]
[20:02:29] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available filters: ["deinterlace_qsv", "deinterlace_vaapi", "hwupload_cuda", "hwupload_vaapi", "libplacebo", "overlay_opencl", "overlay_qsv", "overlay_vaapi", "overlay_vulkan", "overlay_cuda", "procamp_vaapi", "scale_cuda", "scale_opencl", "scale_qsv", "scale_vaapi", "scale_vulkan", "tonemap_cuda", "tonemap_opencl", "tonemap_vaapi", "vpp_qsv", "yadif_cuda", "zscale", "alphasrc"]
[20:02:30] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available hwaccel types: ["cuda", "vaapi", "qsv", "drm", "opencl", "vulkan"]
[20:02:30] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: /usr/lib/jellyfin-ffmpeg/ffmpeg
[20:02:30] [INF] [1] Emby.Server.Implementations.ApplicationHost: ServerId: 90096da0fb9141aebff6ea0f64d5efc4
[20:02:30] [INF] [1] Emby.Server.Implementations.ApplicationHost: Core startup complete
[20:02:30] [INF] [1] Main: Startup complete 0:00:03.5697893
[20:02:32] [INF] [10] Emby.Server.Implementations.ScheduledTasks.TaskManager: Clean up collections and playlists Completed after 0 minute(s) and 0 seconds
[20:02:37] [INF] [14] Emby.Server.Implementations.Updates.InstallationManager: Plugin installed: AniDB 8.0.0.0
[20:02:37] [INF] [14] Emby.Server.Implementations.ApplicationHost: App needs to be restarted.
[20:02:37] [INF] [19] Emby.Server.Implementations.Updates.InstallationManager: Plugin installed: AniSearch 5.0.0.0
[20:02:37] [INF] [19] Emby.Server.Implementations.ApplicationHost: App needs to be restarted.
[20:02:38] [INF] [14] Emby.Server.Implementations.Updates.InstallationManager: Plugin installed: Open Subtitles 20.0.0.0
[20:02:38] [INF] [14] Emby.Server.Implementations.ApplicationHost: App needs to be restarted.
[20:02:38] [INF] [14] Emby.Server.Implementations.Updates.InstallationManager: Plugin installed: TheTVDB 11.0.0.0
[20:02:38] [INF] [14] Emby.Server.Implementations.ApplicationHost: App needs to be restarted.
[20:02:38] [INF] [14] Emby.Server.Implementations.ScheduledTasks.TaskManager: Update Plugins Completed after 0 minute(s) and 5 seconds
```

### Please attach any screenshots here

![image](https://github.com/jellyfin/jellyfin/assets/70137651/73defa95-d23d-4b49-8cf6-d84b4fbe870a)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Inovvia** [comment]
I was able to fix this by deleting my network.xml

**Comment by goratchet766** [comment]
Same issue but deleting network.xml doesn't fix it for me

**Comment by goratchet766** [comment]
So I got Jellyfin working, but base URL support seems to be broken -- at least for me

**Comment by gnattu** [comment]
> So I got Jellyfin working, but base URL support seems to be broken -- at least for me

You need to clear browser cache after setting base url, otherwise the old server setting is still stored in browser's local storage which obviously won't work. I agree we could handle this better but for now this is what you have to do.

**Comment by steffbp** [comment]
> > So I got Jellyfin working, but base URL support seems to be broken -- at least for me
> 
> You need to clear browser cache after setting base url, otherwise the old server setting is still stored in browser's local storage which obviously won't work. I agree we could handle this better but for now this is what you have to do.

Thank you, that fixed it for me.

**Comment by piyushrungta25** [comment]
Had similar issue and can confirm that deleting the network.xml and restarting the service fixed the issue for me. Only network related change I had done was to allow some remote IPs in 10.64.x.x range (so that I can access the server over Tailscale). Things are working flawlessly otherwise.

**Comment by Cl4whammer** [comment]
i deleted the xml file from: 
C:\ProgramData\Jellyfin\Server\config
After restart no new xml file was genereated, so i went to network settings and pressed the save button. After that new xml file generated. But still can only access server with localhost, other pcs cant load the page, even with fresh browsers.

My current xml file:

```
<?xml version="1.0" encoding="utf-8"?>
<NetworkConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <BaseUrl />
  <EnableHttps>false</EnableHttps>
  <RequireHttps>false</RequireHttps>
  <InternalHttpPort>8096</InternalHttpPort>
  <InternalHttpsPort>8920</InternalHttpsPort>
  <PublicHttpPort>8096</PublicHttpPort>
  <PublicHttpsPort>8920</PublicHttpsPort>
  <AutoDiscovery>true</AutoDiscovery>
  <EnableUPnP>false</EnableUPnP>
  <EnableIPv4>true</EnableIPv4>
  <EnableIPv6>false</EnableIPv6>
  <EnableRemoteAccess>true</EnableRemoteAccess>
  <LocalNetworkSubnets />
  <LocalNetworkAddresses />
  <KnownProxies />
  <IgnoreVirtualInterfaces>true</IgnoreVirtualInterfaces>
  <VirtualInterfaceNames>
    <string>veth</string>
  </VirtualInterfaceNames>
  <EnablePublishedServerUriByRequest>false</EnablePublishedServerUriByRequest>
  <PublishedServerUriBySubnet />
  <RemoteIPFilter />
  <IsRemoteIPFilterBlacklist>false</IsRemoteIPFilterBlacklist>
</NetworkConfiguration>
```

**Comment by chiragkrishna** [comment]
having the same issue, only thing which works is to restart jellyfin.service
`sudo systemctl restart jellyfin.service`

but if i restart the system, its the same

**Comment by faisalalsalm** [comment]
i have same issue on window

**Comment by spupuz** [comment]
same problem for me:

[11:54:33] [ERR] [1] Main: Kestrel failed to start! This is most likely due to an invalid address or port bind - correct your bind configuration in network.xml and try again

[11:54:33] [FTL] [1] Main: Error while starting server

**Comment by chiragkrishna** [comment]
issue not resolved in 10.9.1 also, after reboot i have to restart jellyfin.service to make it work

**Comment by gnattu** [comment]
The original issue is about a stale network config, and if that does not fix the issue for you, then this is not the same issue and please file a new one with complete logs and environment info so that we can look at it. 

Closing as the original issue is resolved.

We are having two other known cause issues:

- Instances relying on virtual interfaces named in the `veth*` pattern to connect to outside should set `IgnoreVirtualInterfaces` to `false` as described in #11548, this affects Hyper-V users on Windows most.
- Instances having multiple NICs having same IP as described in #11557, this will be fixed in #11587.

If none of the above applies to you, please, file a new issue. Thank you.

**Comment by daixy2021** [comment]
Emby.Server.Implementations.HttpServer.WebSocketConnection: WS "223.11.142.10" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
